jsonparser.cpp

```diff
@@ -78,9 +122,9 @@ void JsonParser::parseAndSet(const QByteArray &data, const QString label)
         QString projectUrl = itProjectVersions.value().toObject()["url"].toString();
 
         // get projects (imx6, Wetek, ...)
-        QJsonObject val = itProjectVersions.value().toObject()["project"].toObject();
-        for (QJsonObject::Iterator itProjects  = val.begin();
-                                   itProjects != val.end();
+        QJsonObject projectVersionsNode = itProjectVersions.value().toObject()["project"].toObject();
+        for (QJsonObject::Iterator itProjects  = projectVersionsNode.begin();
```

Comment:
same below:
```suggestion
        for (auto itProjects = projectVersionsNode.begin();
```
you can also move all lines of `for` onto a single line